### ISSUE TITLE
Automatically scan the whole playlist.

### ISSUE THREAD

**Issue by Shahim1331** [TeamNewPipe/NewPipeExtractor#1011]
When I open a playlist, only 100 videos get scanned, so when I make the playlist play in background randomly, only that 100 are randomised while the rest come after I complete listening to the 100 songs. To solve this, everytime before I listen, I scroll down and get all the songs scanned and then when I click randomise, all 700+ get randomised. Of course, this isn't an urgent issue, but it would be nice if you could include an option to automatically scan every single file in a playlist without having to go through the manual labour.

In summary, I'd like an option to scan all the files in a playlist every time I open it or save the complete playlist to the cache so that I have to open and scroll down only once.
